apps/vr-tests-web-components/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@fluentui/vr-tests-web-components",
+  "version": "0.0.1",
+  "private": true,
+  "description": "Visual regression tests for @fluentui/web-components",
+  "type": "module",
+  "scripts": {
+    "build": "build-storybook -o dist/storybook",
+    "format": "prettier . -w --ignore-path ../../.prettierignore",
+    "lint": "eslint src --ext .ts,.tsx",
+    "start": "start-storybook",
+    "test": "jest --passWithNoTests",
+    "type-check": "tsc",
```

Comment:
follow up: this needs to use our `type-check` otherwise we will run into same perf issues as in the past

Reply:
@chrisdholt can you file an issue to track follow up?

Reply:
> @chrisdholt can you file an issue to track follow up?

Sure thing - I was just looking at removing this but it looks like this is aligned with the react VR tests, so I'll create an issue for follow-up since it's not scoped to just our package.